domain/access/state/permission.go

```diff
@@ -194,28 +215,54 @@ func (st *PermissionState) ReadUserAccessForTarget(ctx context.Context, subject
 		return userAccess, errors.Trace(err)
 	}
 
+	type input struct {
+		Name    string `db:"name"`
+		GrantOn string `db:"grant_on"`
+	}
+
+	readQuery := `
+SELECT  (p.uuid, p.grant_on, p.grant_to, p.access_type) AS (&dbReadUserPermission.*),
+        (u.uuid, u.name, u.display_name, u.created_at, u.disabled) AS (&dbPermissionUser.*),
+        creator.name AS &dbPermissionUser.created_by_name
+FROM    v_user_auth u
+        LEFT JOIN user AS creator ON u.created_by_uuid = creator.uuid
```

Comment:
`created_by` is `NOT NULL`. Given that it is always there and there is RI between these two, this can be a simple inner `JOIN`. The record is always there.

Reply:
done